library/connect-go/v0.4.0/buf.plugin.yaml

```diff
@@ -4,12 +4,12 @@ plugin_version: v0.4.0
 source_url: github.com/bufbuild/connect-go
 description: A better gRPC.
 deps:
-  - plugin: buf.build/library/go:v1.28.0
+  - plugin: buf.build/library/go:v1.28.1
 default_opts:
   - paths=source_relative
 registry:
   go:
     min_version: "1.18"
     deps:
       - module: github.com/bufbuild/connect-go
-        version: v0.3.0
+        version: v0.4.0
```

Comment:
Updated this (it would be good to update our versions script to set this too).

Reply:
For the plugins we manage, such as `connect-go` and `connect-web`, the plugin version has parity with the runtime.

But I wonder how this would play out for the others and whether we could do it programatically, e.g., `protoc-gen-go-grpc` ?

The plugin version is `v1.2.0` but the dependency is `v1.43.0` (we should probably bump it to `v1.48.0`).